modules/@apostrophecms/i18n/index.js

```diff
@@ -687,7 +687,7 @@ module.exports = {
         }
         return i18n;
       },
-      getLocales() {
+      async getLocales() {
```

Comment:
not sure why this or verifyLocales is async?

Reply:
no longer async